diagnostic.md

```diff
@@ -5,20 +5,29 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    A series of lists, each of which has several items, can be modeled as two
+    components: one for the lists, and one for the list items
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    app/component/component-name/component.js: this file determines the
+    attributes for the component, like what tag surrounds the component
+    (e.g. 'ul') and what actions the component can perform
+
+    app/component/component-name/template.hbs: this file determines how the
+    component will be rendered on the page
```

Comment:
Yes.  Template files are responsible for containing markup
(defining structure), binding data, and binding and dispatching actions.